src/helm/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to
 
 ## [Unreleased]
 
+## [0.3.0] - 2024-02-07
```

Comment:
A footer link for this release is missing!

Reply:
Whoopsie, fixed